bank_integration/bank_integration/api/hdfc_bank_api.py

```
@@ -848,160 +941,319 @@ def payment_success(self):
+            "xpath",
+        )
+        file_format_selector.click()
 
-                bank_transaction = frappe.get_doc({"doctype": "Bank Transaction"})
+        try:
+            download_btn = self.get_element(
+                "//button[contains(@class,'download-button') and normalize-space()='Download']",
+                "xpath",
+            )
+            download_btn.click()
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)